src/InvisibleReCaptcha/InvisibleReCaptchaInput.php

```diff
@@ -25,12 +34,12 @@ public function __construct(\Contributte\ReCaptcha\ReCaptchaProvider $provider,
 		$this->provider = $provider;
 		$this->form = $form;
 		$this->versioningProvider = $versioningProvider;
+		$this->contentSecurityPolicy = $contentSecurityPolicy;
+
+		$this->setNonce();
```

Comment:
Osobně bych to nedával ani do metody, protože nemá žádné vstupy a výstupy, jen jednu podmínku. Stejně tak `$this->contentSecurityPolicy` se už nikde jinde nepoužívá. Až bude pouze PHP 8, tak by tu bylo `$this->nonce = $contentSecurityPolicy?->getNonce();`